src/main/resources/application.yml

```diff
@@ -0,0 +1,21 @@
+server:
+  port: 0
```

Comment:
Why did you add server.port property?

Reply:
Because it tries to use port 8080 by default if I don't add it. And I have the management app running on port 8080.